### PR TITLE
UsesOfSymbolAtPointReq, do not try to solve symbols from classfiles (#1827)

### PR DESCRIPTION
**Pull request by hzenginx**
I am not sure if there is a way to find usages of a symbol coming from classfiles, Intellij IDEA CE is also not able to do this.
This pr only prevents NPEs and returns a empty list as usages.

UsesOfSymbolAtPointReq, do not try to solve symbols from classfiles (#1827)

### CODE REVIEW

File: core/src/it/scala/org/ensime/intg/BasicWorkflow.scala [hunk 1]
```diff
@@ -146,6 +146,20 @@ class BasicWorkflow extends EnsimeSpec
...
           }
 
+          //-----------------------------------------------------------------------------------------------
+          // uses of symbol at point with a symbol defined in classfiles
+
+          project ! TypecheckFilesReq(List(Left(fooFile)))
+          expectMsg(VoidResponse)
+
+          project ! UsesOfSymbolAtPointReq(Left(fooFile), 162) // point on String.length
+          expectMsgType[ERangePositions].positions should contain theSameElementsAs List()
```
Comment (fommil) [inline core/src/it/scala/org/ensime/intg/BasicWorkflow.scala]: `should be empty` is cleaner

File: core/src/it/scala/org/ensime/intg/BasicWorkflow.scala [hunk 2]
```diff
@@ -146,6 +146,20 @@ class BasicWorkflow extends EnsimeSpec
...
+
+          asyncHelper.fishForMessage() {
```
Comment (fommil) [inline core/src/it/scala/org/ensime/intg/BasicWorkflow.scala]: we've found these `fishForMessage` waiting blocks to be very unreliable. Please try to work out exactly what messages are being returned and wait for them. It is probably safer to put your test at the very end.

Comment (hzenginx) [inline core/src/it/scala/org/ensime/intg/BasicWorkflow.scala]: This test is really hard to extend :/ I moved the new code to end of the file

**Comment by hzenginx** [comment]
Sure I will try to find all messages. Build is already failed so It makes sense to really figure out all messages 

I am interested in doing that but first I want to merge this because I am using emacs+ensime as  my primary environment at my job and this bug is annoying :P

**Comment by fommil** [comment]
Yeah, I leave bugs in intentionally to trick people into contributing muahahaha!

(Did you see the instructions to use your own build? Really useful for when you have a fix in PR but not merged yet)

**Comment by hzenginx** [comment]
Looks like your plan is working well, this is the second time I got trapped 😄 
I am using my build currently BTW but nothing is better than having it in regular ways.

I added another expected message, hopefully this should work but this test is too complicated to change :(

**Comment by hzenginx** [comment]
Is AppVeyor broken?

**Comment by fommil** [comment]
Yeah... it's increasingly getting stuck at the serverstartspec.

**Comment by hzenginx** [comment]
Should I push some changes just to restart that one, otherwise the tests looks pretty fine to me now

**Comment by fommil** [comment]
Thanks! But damn I meant to squash merge.

**Pull request merged**